### PR TITLE
Moved from UiBinder to Errai UI

### PR DESCRIPTION
**Pull request by pefernan**
- Moved from UiBinder components to Errai UI
- Removed unused constants
- Fixed ClassCastException
- Removed _ on translation keys

### CODE REVIEW

File: livespark-common-widgets/livespark-crud-component/src/main/java/org/livespark/widgets/crud/client/resources/i18n/CrudComponentConstants.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+ */
+
+package org.livespark.widgets.crud.client.resources.i18n;
+
+import org.jboss.errai.ui.shared.api.annotations.TranslationKey;
+
+public interface CrudComponentConstants {
+
+    @TranslationKey( defaultValue = "")
+    String CrudComponentViewImpl_newInstanceButton = "CrudComponentViewImpl.newInstanceButton";
```
Comment (mbarkley) [inline livespark-common-widgets/livespark-crud-component/src/main/java/org/livespark/widgets/crud/client/resources/i18n/CrudComponentConstants.java]: Using underscores in identifiers is deprecated in Java 8 (and will show warnings in the build). I see why you did it in this case, but I think it's best to remove them.

**Comment by mbarkley** [comment]
Hi @pefernan 

As I mentioned in-line, I think we should get rid of the underscores in the field names of the translation key constants since they will cause build warnings.

Otherwise looks good.

**Comment by pefernan** [comment]
@mbarkley tbh I thought it was only with variables called "_" but you're right... so removed them.

**Comment by mbarkley** [comment]
Jenkins, retest this.

**Comment by mbarkley** [comment]
+1

**Pull request merged**